packages/components/src/OverlayTrigger/OverlayTrigger.component.js

```diff
@@ -105,6 +110,7 @@ export default class OverlayTrigger extends React.Component {
 					theme['tc-action-button-positionned'],
 					'tc-action-button-positionned',
 				)}
+				{...tooltipProps}
 			>
 				<BaseOverlayTrigger {...props}>{this.props.children}</BaseOverlayTrigger>
```

Comment:
You pass the mouseOver/Out to the span and the button

Reply:
fixed